config/manager/kustomization.yaml

```diff
@@ -2,7 +2,3 @@ resources:
 - manager.yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-images:
-- name: controller
-  newName: quay.io/cephcsi/ceph-csi-operator
-  newTag: latest
```

Comment:
Lets not remove it from here.

Reply:
We cannot, changes are cascading which means any patch here "overwrites" any patch coming from a higher overlay. I was chasing this bug almost 1.5 hours before I saw you committed this change some time ago.
As a rule of thumb, if we need customization on top of any persistent repo file it should come from an overlay.

Reply:
We dont need an higher overlay at all IMO.

Reply:
I would say the opposite! we should not touch any of the existing bases/overlays with local changes

Reply:
As this is not about the correctness of the solution but about the approach (which is somewhat a matter of opinion),  I am resolving this so we could merge, to unblock us. We can continue the discussion after merging and if needed we can always pivot with a new PR on top